shell/platform/android/io/flutter/embedding/android/FlutterActivityAndFragmentDelegate.java

```diff
@@ -87,6 +89,7 @@
         public void onFlutterUiDisplayed() {
           host.onFlutterUiDisplayed();
           isFlutterUiDisplayed = true;
+          isFirstFrameRendered = true;
```

Comment:
nit: can `isFlutterUiDisplayed` be used instead?

Reply:
That flips to false if the ui goes away